microsoft-edge/devtools-guide-chromium/workspaces/index.md

```diff
@@ -22,145 +22,168 @@ ms.date: 05/04/2021
 
-1. Open `styles.css`.  The `color` property of `h1` elements is set to `fuchsia`.
+The following steps show how to make a change in the CSS file and save it to disk.
+
+1. Open `styles.css`.  The `color` property of the `h1` element is set to `fuchsia`.
 
    ![View styles.css in a text editor.](../media/workspaces-workspaces-demo-sources-filesystem-css.msft.png)
 
 1. Select the **Elements** tool.
 
-1. Change the value of the `color` property of the `<h1>` element to your favorite color.  To do this, select the `<h1>` element in the **DOM Tree**.
+   The CSS rules that are applied to the `<h1>` element are shown in the **Styles** pane.  The page indicator next to `styles.css:1` means that any changes that you make are mapped to `~/Desktop/app/styles.css`.
+
+   ![The indicator that the file is linked.](../media/workspaces-workspaces-demo-elements-styles-css.msft.png)
 
-   The CSS rules that are applied to the `<h1>` element are shown in the **Styles** pane.  The green dot next to `styles.css:1` means that any change that you make are mapped to `~/Desktop/app/styles.css`.
+1. Change the value of the `color` property of the `<h1>` element to your favorite color.  To do this, select the `<h1>` element in the **DOM Tree**. Select `fucshia`, type the new color, and then select it from the color list shown in the next screenshot.
 
-   ![The green indicator that the file is linked.](../media/workspaces-workspaces-demo-elements-styles-css.msft.png)
+   ![Change the color property in styles.css.](../media/workspaces-workspaces-demo-elements-styles-css-pick-color.msft.png)
 
-1. Open `styles.css` in a text editor again.  The `color` property is now set to your favorite color.
+1. Open `styles.css` in a text editor.  The `color` property is now set to the new color, which is orange in this example.
 
 1. Refresh the page.
 
-The color of the `<h1>` element is still set to your favorite color.  The change remains across a refresh, because when you made the change DevTools saved the change to disk.  And then, when you refreshed the page, your local server served the modified copy of the file from disk.
+The color of the `<h1>` element is still set to the new color.  The change remains across a refresh, because when you made the change, DevTools saved the change to disk. When you refreshed the page, your local server served the modified copy of the file from disk.
 
+**Tip:** You can also change the color by clicking the fucshia-colored swatch to open the color picker to pick a new color. The HEX value for the color you pick is the color name.
 
 <!-- ====================================================================== -->
-## Step 3: Save an HTML change to disk
+#### Save an HTML change to disk
 
-You can change HTML tagging using the Elements tool, but to be able to save the edits, we'll use the Sources tool.
+You can change HTML tagging using the **Elements** tool, but if you want to save your edits you need to use the **Sources** tool.
```

Comment:
```suggestion
In the **Elements** tool, it's possible to change HTML tagging in a copy of the file that's returned by the server.  However, to save your edits to a local source file, you need to use the **Sources** tool instead of the **Elements** tool.
```